src/lib/eliom_client.client.ml

```diff
@@ -70,8 +70,13 @@ let get_global_data () =
     None
 
 let init_client_app
-    ~app_name ?(ssl = false) ~hostname ?(port = 80) ~full_path () =
+    ?app_path ~app_name
```

Comment:
I am still unsure about `full_path`. It is not documented anywhere. It is stored in `Eliom_common_base.client_process.info`, field `cpi_original_full_path`. Do you happen to remember what that one is for? If it is for the path of the app (quite plausible) I can use it.